apps/accounts/tests/test_views.py

```diff
@@ -517,7 +518,7 @@ def test_identify_success_no_account(self):
         response = self.client.get(
             self.url, {'code': 'codes!!', 'state': 'the-right-blob'})
         assert response.status_code == 302
-        assert response['location'] == 'http://testserver/'
+        assert response['location'] == absolutify(reverse('users.edit'))
```

Comment:
did you consider [assertRedirects()](https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.SimpleTestCase.assertRedirects)? The exception message is pretty good.